### ISSUE TITLE
Default nAttrMon hbs template failing for Email notification

### ISSUE THREAD

**Issue by LeandroFernandes5** [OpenAF/nAttrMon#154]
**Describe the bug**
When using the default email template for email warnings we see the attached error in the logs. This prevents the email from being sent.

**To Reproduce**
1. Create fake email using default email template provided by nAttrMon

**Screenshots**
Attached
![MicrosoftTeams-image](https://user-images.githubusercontent.com/32273119/234134420-4160e3b2-e5fe-4364-bd90-0763e9f8c68f.png)

**Comment by nmaguiar** [comment]
Hi @LeandroFernandes5,

Wasn't this solved already [on commit 84abf8ceaae60e2c65968cbfa9d940e46ee2f6db](https://github.com/OpenAF/nAttrMon/commit/84abf8ceaae60e2c65968cbfa9d940e46ee2f6db#diff-5ee01cf66e704e701d0b659a7821495c3a7228de43f8902d8d165fe097353d74)?

You can do a simple test:

````javascript
> tprint("{{#$is level \"High\"}}red{{/$is}}", { level: "High" })
red
> tprint("{{#is level \"High\"}}red{{/is}}", { level: "High" })
Error: Missing helper: "is"
````

The generic email template was changed in order to be compatible with a breaking change on [OpenAF 20220822](https://github.com/OpenAF/openaf/releases/tag/20220822).

Can you check if that is the issue?

Thank you,
Nuno Aguiar

**Comment by LeandroFernandes5** [comment]
Hi @nmaguiar,

It seems they are using the latest nAttrMon but that file hasn't been updated. Assuming that something went wrong during the upgrade steps.

Closing the issue.

Thank you!
